test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -49,6 +51,19 @@
     String retention = "1000";
     String compressionCodec = Codec.Type.LZ4.toString();
 
+    boolean noCompaction = true;
+    String compactionPolicyType = "SNAPSHOT_LENGTH_FIRST";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651450)